jobs/build/appregistry/Jenkinsfile

```diff
@@ -132,25 +149,35 @@ node {
                     return
                 }
 
-                currentBuild.description += "\npushing operator metadata."
-                withCredentials([usernamePassword(credentialsId: 'quay_appregistry_omps_bot', usernameVariable: 'QUAY_USERNAME', passwordVariable: 'QUAY_PASSWORD')]) {
-                    def token = retrieveBotToken()
-                    for (def i = 0; i < operatorData.size(); i++) {
-                        def build = operatorData[i]
-                        retry(3) {
-                            def response = httpRequest(
-                                url: "https://omps-prod.cloud.paas.psi.redhat.com/v2/redhat-operators-art/koji/${build.nvr}",
-                                httpMode: 'POST',
-                                customHeaders: [[name: 'Authorization', value: token]],
-                                timeout: 60,
-                                validResponseCodes: "200:599",
-                            )
-                            if (response.status != 200) {
-                                sleep(5)
-                                error "OMPS request for ${build.nvr} failed: ${response.status} ${response.content}"
+                if (params.STREAM == 'dev') {
+                    currentBuild.description += "\npushing operator metadata."
+                    withCredentials([usernamePassword(credentialsId: 'quay_appregistry_omps_bot', usernameVariable: 'QUAY_USERNAME', passwordVariable: 'QUAY_PASSWORD')]) {
+                        def token = retrieveBotToken()
+                        for (def i = 0; i < operatorData.size(); i++) {
+                            def build = operatorData[i]
+                            retry(3) {
+                                def response = httpRequest(
+                                    url: "https://omps-prod.cloud.paas.upshift.redhat.com/v2/redhat-operators-art/koji/${build.metadata_nvr}",
```

Comment:
this says prod. is that expected?

Reply:
apparently yes ... the endpoint is always that one (omps-prod) ... what can change is our namespace, but so far `redhat-operators-art` was the only one I could find.

Reply:
sounds weird. but ok.